src/Form/Form.js

```diff
@@ -410,10 +449,14 @@ class Form {
 
         // If the form has too many inputs, destroy everything to avoid performance issues
         if (this.inputs.all.size > MAX_INPUTS_PER_FORM) {
-            if (shouldLog()) {
-                console.log('The form has too many inputs, destroying.')
-            }
-            this.destroy()
+            this.device.scanner.stopScanner('The form has too many inputs, destroying.')
```

Comment:
Again, going nuclear. Instead of just `destroy`ing the form, we stop the scanner altogether.